applications/daily_tests/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "endOfLine": "auto"
```

Comment:
Is this a problem on windows? Mixing line endings between files does not seem like a good idea, preferably any editor regardless of OS should be fine with `lf` line endings. Our policy is to use prettier defaults so that we avoid bike shedding.

Reply:
Without it on windows the prettier reformats the whole file. The LF endings are forced by git settings anyway, so this should not affect the commits. But without this setting the editing on windows mess up the compare during development. By the way I copied this form the integration folder.

Reply:
Ah ok makes sense